src/util/language.cpp

```diff
@@ -8,6 +8,10 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "language.h"
 #include "expr.h"
+#include <util/config.h>
+#include <util/symbol.h>
+#include <util/symbol_table.h>
+#include <goto-programs/goto_functions.h>
```

Comment:
This seems scary: util should not depend on goto programs in any way. Also, the other headers should just be "config.h" etc.